src/components/Product/SingleProduct.tsx

```diff
@@ -139,7 +139,7 @@ const SingleProduct = (props: any) => {
 							{highestBid ?
 								<h4 className="prod-price">Start from <span>${highestBid}+</span></h4> : ''
 							}
-							{loggedUser ?
+							{loggedUser && user?.email !== item?.seller.email ?
```

Comment:
Do you have the id available when you check this? Would be a more robust check as I think it's conceivable that a user would change their email address within the system, but they cannot change their id.